azure-pipelines/build/linux/du/templates/dopapt-docker-steps.yml

```diff
@@ -29,11 +37,51 @@ steps:
     script: 'sudo docker run --rm --privileged multiarch/qemu-user-static --reset -p yes'
   displayName: Enable the build VM to run multiarch docker container
 
+# - task: CmdLine@2
+#   condition: eq('${{parameters.skipTests}}', false)
+#   inputs:
+#     script: 'sudo dpkg -i /tmp/build-deliveryoptimization-plugin-apt-${{parameters.targetOsArch}}/linux-${{parameters.config}}/deliveryoptimization-agent*.deb'
+#     workingDirectory: '/tmp/build-deliveryoptimization-plugin-apt-${{parameters.targetOsArch}}/linux-${{parameters.config}}'
+#   displayName: 'Install agent Debian package'
+
+- task: CmdLine@2
+  inputs:
+    script: 'python3 ./build.py --project sdk --cmaketarget deliveryoptimization --config ${{parameters.config}} --package-for DEB --clean'
```

Comment:
>python3 [](http://example.com/codeflow?start=13&length=7)

The sdk build must happen inside the container, so bring the command from the sdk steps file.

You will have to adjust the path in "Install libdeliveryoptimization" step after that.